set_theia_version.sh

```diff
@@ -71,8 +71,6 @@ change_change_che_theia_init_baranch () {
         fi
     done < "che-theia-init-sources.yml";
     printf "%b\n" "$content" > "che-theia-init-sources.yml";
-
-    sed -i 's~@api.github.com/repos/eclipse/che-theia/git/${GIT_REF}~@api.github.com/repos/eclipse/che-theia/git/refs/heads/'${branchName}'~' ./dockerfiles/theia/Dockerfile 
```

Comment:
This edit actually does nothing as there no such command in the target file.
The correct value for `GIT_REF` env. variable comes from `build.include`, see the corresponding sed edit
https://github.com/eclipse/che-theia/blob/8fc54d6e8629d7fd405ea243498310c961e050b8/set_theia_version.sh#L128